src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -345,6 +361,7 @@ def connect_to_fluent(
     :class:`~ansys.fluent.core.session_solver_icing.SolverIcing`]
         Session object.
     """
+    _check_ip_port(ip=ip, port=port)
```

Comment:
@hpohekar I belive that if we are to do this (which seems like a reasonable idea) then we should move it one line down to catch the scenario where an out-of-date server file is specified.

Reply:
@seanpearsonuk Yes, I agree. Actually this should be inside `_get_server_info`. We have moved it there. Thank you.